mobile/src/main/res/values-de/strings.xml

```diff
@@ -249,10 +249,10 @@
     <!-- Device identifier -->
     <string name="device_identifier_title">Gerätekennung</string>
     <string name="device_identifier_prefix_voice"><![CDATA[Sprachbefehlen das Präfix \'<Gerätekennung>|\' voranstellen]]></string>
-    <string name="device_identifier_bg_tasks">Nutze dies als Präfix für die Item-Namen von \"Geräteinformationen an Server senden\"</string>
+    <string name="device_identifier_bg_tasks">Als Präfix für die Item-Namen von \"Geräteinformationen an Server senden\" verwenden</string>
     <string name="device_identifier_hint">Die Gerätekennung kann jede beliebige Zeichenkette sein und sollte für alle Geräte, die auf Ihren openHAB Server zugreifen, einzigartig sein. Neben den beiden oben genannten Anwendungsfällen, kann die Gerätekennung auch als Status bei NFC-Tags verwendet werden, sodass Sie wissen, welches Gerät den Tag liest.</string>
     <string name="device_identifier_suggested_command_nfc_tag">Gerätekennung des scannenden Gerätes (\"%s\" für dieses Gerät)</string>
-    <string name="device_identifier_summary_not_set">Daten, die an den Server gesendet werden, und Sprachbefehle haben nicht die Gerätekennung als Präfix</string>
+    <string name="device_identifier_summary_not_set">Für Daten, die an den Server gesendet werden, und Sprachbefehle wird kein gerätespezifischer Präfix verwendet</string>
```

Comment:
```suggestion
    <string name="device_identifier_summary_not_set">Für Daten, die an den Server gesendet werden, und Sprachbefehle wird kein gerätespezifisches Präfix verwendet</string>
```
(@maniac103 Präfix ist neutrum, daher "gerätespezifisches" https://www.duden.de/rechtschreibung/Praefix Ich ändere es in Crowdin.)